dnsprovider/google.go

```diff
@@ -22,35 +22,124 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 
+	"golang.org/x/oauth2/google"
 	"google.golang.org/api/dns/v1"
+	googleapi "google.golang.org/api/googleapi"
```

Comment:
You are right, I don't know if goimports did this or I just copied from somewhere. I can change if you like.